flake.nix

```diff
@@ -0,0 +1,448 @@
+{
+  description = "BDK Flake to run all tests locally and in CI";
+
+  inputs = {
+    # stable nixpkgs
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.05";
+    # pin bitcoind to a specific version
+    # find instructions here:
+    # <https://lazamar.co.uk/nix-versions>
+    # pinned to 0.25.0
+    nixpkgs-bitcoind.url = "github:nixos/nixpkgs?rev=9957cd48326fe8dbd52fdc50dd2502307f188b0d";
+    # Blockstream's esplora
+    # inspired by fedimint CI
+    nixpkgs-kitman.url = "github:jkitman/nixpkgs/add-esplora-pkg";
+    crane = {
```

Comment:
**suggestion:** Would also lock `crane` to some release commit. Updating it is not critical, and regressions do happen in the master branch.

Reply:
Good suggestion. I am pinning it